ui.apps/src/main/content/jcr_root/apps/hclecomm/components/plugins/clientlibs/js/color-picker.js

```diff
@@ -0,0 +1,378 @@
+// RTE ColorPicker Plugin code
+(function ($, CUI) {
+    var GROUP = "colorformat",
+        COLOR_PICKER_FEATURE = "colorPicker",
+        DIALOG = "colorPickerDialog",
+        PICKER_NAME_IN_POPOVER = "color",
+        REQUESTER = "requester",
+        TITLE = "Color Picker",
+        ICON = "textColor",
+        PICKER_URL = "/apps/hclecomm/components/plugins/popovers/color-picker/_cq_dialog.html";
```

Comment:
@nitheshkumar-hcl : any other chnage apart from this line in the JS ?

Reply:
no other change apart from the picker URL  change in js file.